app/models/concerns/orchestration/compute.rb

```diff
@@ -106,7 +106,15 @@ def setCompute
     final_compute_attrs = compute_attributes.merge(compute_resource.host_compute_attrs(self))
     self.vm = compute_resource.create_vm(final_compute_attrs)
   rescue => e
-    failure _("Failed to create a compute %{compute_resource} instance %{name}: %{message}\n ") % { :compute_resource => compute_resource, :name => name, :message => e.message }, e
+    # Workaround bug when the exception itself contains unresolved string placeholder
```

Comment:
Is this really needed if we solve the original bug?

Reply:
imho yes because such bug could easily be reintroduced somewhere and it was not easy to track at all. Having the original untranslated message with unresolved placeholder would have pointed me to the root cause immediately...